packer/build_image.sh

```diff
@@ -293,8 +293,9 @@ elif [ "$TARGET" = "azure" ]; then
     PACKER_ARGS+=(-var subscription_id="$AZURE_SUBSCRIPTION_ID")
 fi
 
+IMAGE_NAME="$PRODUCT-$VERSION-$ARCH-$(date "+%FT%T")"
```

Comment:
I guess the double quote in the command substitution should be changed to a single quote (the " ends the line)
Other than that, lgtm